tests/conftest.py

```diff
@@ -13,75 +17,40 @@ def tmpdir(tmp_path_factory):
 
 
 @pytest.fixture(scope="session")
-def fastspec_data():
-    ntimes = 2
-    nfreqs = 32768
-
-    attrs = {
-        "fastspec_version": "0.0.0",
-        "start": 0,
-        "stop": 0,
-        "site": "simulated",
-        "instrument": "mid",
-        "switch_delay": 0.5,
-        "input_channel": 1,
-        "voltage_range": 0,
-        "samples_per_accumulation": 4294967269,
-        "acquisition_rate": 400,
-        "num_channels": nfreqs,
-        "num_taps": 5,
-        "window_function_id": 3,
-        "num_fft_threads": 10,
-        "num_fft_buffers": 100,
-        "stop_cycles": 0,
-        "stop_seconds": 0.0,
-        "stop_time": "",
-    }
-
-    spec = {
-        "p0": np.zeros((ntimes, nfreqs)),
-        "p1": np.zeros((ntimes, nfreqs)),
-        "p2": np.zeros((ntimes, nfreqs)),
-        "Q": np.zeros((ntimes, nfreqs)),
-    }
+def edgeslow():
+    return KNOWN_TELESCOPES["edges-low"]
 
-    fq_anc = {"frequencies": np.linspace(40, 200, nfreqs)}
-    time_anc = {
-        "times": np.array(["2020:001:01:01:01", "2020:001:01:02:01"], dtype="|S17"),
-        "adcmax": np.zeros((ntimes, 3)),
-        "adcmin": np.zeros((ntimes, 3)),
-        "data_drops": np.zeros((ntimes, 3), dtype="int"),
-    }
 
-    return {
-        "meta": attrs,
-        "spectra": spec,
-        "time_ancillary": time_anc,
-        "freq_ancillary": fq_anc,
-    }
+@pytest.fixture(scope="session")
+def small_gsdata_obj(edgeslow: Telescope):
+    ntimes = 2
+    nfreqs = 32768
+    npols = 1
+    nloads = 3
```

Comment:
is `nloads` loads as we know it? there are 4 of them!

Reply:
`loads` is, alas, a loaded term...

There are 4 calibration "loads". Here, it refers to the three internal "loads" with which we perform the 3-position switch calibration (i.e. antenna, load, load+noise source). Probably, this is bad terminology. But yes, it's meant to be 3.